### PR TITLE
Changed documentation of getMax function in radix.cpp

### PR DESCRIPTION
**Pull request by SamSaver**
Samarth Singh Pawar
11941060
samarths@iitbhilai.ac.in

### CODE REVIEW

File: Sorting/radix.cpp [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
         ret *= base;
-    
+
     return ret;
 }
 
 /**
- * Documentation should go here
+ * This function returns the maximum value from the input vector.
+ * Input Parameter: std::vector<int> input
```
Comment (shashwatj07) [inline Sorting/radix.cpp]: ```Inputs: std::vector<int> input```

File: Sorting/radix.cpp [hunk 2]
```diff
@@ -5,15 +5,17 @@
...
+ * Return Value: int maxVal (This is the maximum element in the input vector)
```
Comment (shashwatj07) [inline Sorting/radix.cpp]: ```Returns: The maximum value in the input vector```

File: Sorting/radix.cpp [hunk 3]
```diff
@@ -5,15 +5,17 @@
...
     f(i,exp)
         ret *= base;
-    
+
     return ret;
 }
 
 /**
- * Documentation should go here
+ * This function returns the maximum value from the input vector.
```
Comment (shashwatj07) [inline Sorting/radix.cpp]: ~from~ in

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
Some items

**Comment by shashwatj07** [comment]
Comment "done" on each, after you push.

File: Sorting/radix.cpp (hunk 1 above)
Comment (SamSaver) [reply Sorting/radix.cpp]: Done.

File: Sorting/radix.cpp (hunk 2 above)
Comment (SamSaver) [reply Sorting/radix.cpp]: Done.

File: Sorting/radix.cpp (hunk 3 above)
Comment (SamSaver) [reply Sorting/radix.cpp]: Done.

**Review by shashwatj07 — APPROVED** [review]
LGTM

**Review by yatendra1999 — APPROVED** [review]
LGTM

**Pull request merged**